packages/expo-screen-orientation/src/ExpoScreenOrientation.web.ts

```diff
@@ -33,9 +32,9 @@ const OrientationWebToAPI: {
 
 declare const window: Window;
 
-const { screen } = canUseViewport && window;
+const { screen }: any = Platform.canUseViewport && window;
```

Comment:
I don't like `any` type here. Can we change it to something else?

Reply:
yeah, updated here https://github.com/expo/expo/pull/11398/commits/51a0b5d5494de3712fe8985568d37b5e4ae3093e